akka-http2-support/src/main/scala/akka/http/impl/engine/http2/Http2Multiplexer.scala

```diff
@@ -231,7 +231,7 @@ private[http2] trait Http2MultiplexerSupport { logic: GraphStageLogic with Stage
         state = nextState
       }
 
-      sealed trait MultiplexerState extends Product {
+      private[http2] sealed trait MultiplexerState extends Product {
```

Comment:
To help the Scala compiler notice that nobody will see this class outside `http2`. It complained about:
> method pushControlFrame in class WaitingForConnectionWindow references private[http2] trait FrameEvent.

Reply:
Ok, if that helps with that.